<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadpro</class>
 <widget class="QWidget" name="cadpro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Produtos</string>
  </property>
  <widget class="QLabel" name="labPreco">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>61</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labFabricante">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>61</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fabricante</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPreco">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtFabricante">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCodigo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDescricao">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labDescricao">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição</string>
   </property>
  </widget>
  <widget class="QLabel" name="labCodigo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>33</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Código</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnIncluir">
      <property name="text">
       <string>&amp;Incluir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAlterar">
      <property name="text">
       <string>&amp;Alterar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLocalizar">
      <property name="text">
       <string>&amp;Localizar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExcluir">
      <property name="text">
       <string>&amp;Excluir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFechar">
      <property name="text">
       <string>&amp;Fechar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
